style.css

```diff
@@ -5,6 +5,35 @@
 		Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
 }
 
+.mobile-nav {
+	display: block;
+}
+
+.mobile-nav ul {
+	padding: 0;
+	border: 1px solid;
+	border-radius: 30px;
+	width: 50%;
+	margin: 0 auto 20px;
+	background: rgb(54, 54, 54);
+}
+
+.mobile-nav ul li {
+	list-style: none;
+	margin: 10px 0;
+	text-align: center;
+}
+
+.mobile-nav ul li a {
+	color: white;
+	font-size: 1.2rem;
+	text-decoration: none;
+}
+
+.mobile-nav {
```

Comment:
Duplication of classes. Fix this in accordance with the comment written for myFunction in Js